.vib/airflow/cypress/cypress/integration/airflow_spec.js

```diff
@@ -4,57 +4,31 @@ import { random } from '../support/utils';
 it('allows triggering execution of a sample DAG', () => {
   cy.login();
   cy.fixture('DAGs').then((dags) => {
-    const TRIGGERED_OK_MESSAGE = `Triggered ${dags.triggered.id}, it should start`;
-
     cy.visit(`dags/${dags.triggered.id}/grid`);
-    cy.get('a[aria-label="Trigger DAG"]').click();
+    cy.get('[aria-label="Trigger DAG"]').click();
     cy.contains('button', 'Trigger DAG').click(); // A dropdown appears and clicking again is needed
-    cy.get('.alert-message').should('contain.text', TRIGGERED_OK_MESSAGE);
-  });
-
-  // Verify the DAG appears in the list of active jobs
-  cy.visit('home?status=active');
-  cy.get('span[class="switch"]').should('have.length', 1);
-});
+    cy.contains('div', `Triggered ${dags.triggered.id}`);
```

Comment:
You can safely remove this assertion as we are checking that the DAG appears as active in the next line.